acceptance/cases/models/interleave_test.rb

```diff
@@ -31,6 +32,42 @@ def teardown
       def test_with_partial_inserts_disabled
         AlbumPartialDisabled.create! title: "Title3", singer: singer
       end
+
+      class StringParentKeyTest < SpannerAdapter::TestCase
+        include TestHelpers::WithSeparateDatabase
+
+        class Singer < ActiveRecord::Base
+          has_many :albums, foreign_key: :singer_id
+        end
+
+        class Album < ActiveRecord::Base
+          self.primary_keys = [:singer_id, :album_id]
+          belongs_to :singer, foreign_key: :singer_id
+        end
+
+        def setup
+          super
+
+          connection.ddl_batch do
+            connection.create_table :singers, id: false do |t|
+              t.string :singer_id, limit: 36, primary_key: true, null: false
+              t.string :name, null: false
+            end
+
+            connection.create_table :albums, id: false do |t|
+              t.interleave_in :singers
+              t.string :singer_id, limit: 36, parent_key: true, primary_key: true, null: false
+              t.integer :album_id, primary_key: true, null: false
+              t.string :title, null: false
+            end
+          end
+        end
+
+        def test_create_album
```

Comment:
Would you mind adding a couple of tests for other basic operations as well:
* UpdateAlbum
* GetAlbum

And also verify that it is possible to get the albums of a singer (that is; iterate over the collection of albums of a singer)